oslabs-beta/apollo11
# Apollo tab

**Feature:** Created Initial Layout For The Apollo Tab

**Description:**
- Created initial grid layout for the Apollo Tab
- 3 columns that each take up a third of the Content part of the Mini Drawer component
- Button that toggles the visiblity of the Cache Details tab

**Changes I Made:**
How to Test:
-Go to the third tab from the top and confirm the display has three columns
-Click the button within the Current Cache column and confirm that the Cache Details section toggles its visibility
